Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gableroux/flutter:v1.7.6
+
+WORKDIR /app
+
+COPY ./app_feup/ .
+
+RUN flutter pub get
```

Comment:
Assuming this is like `npm install`

Reply:
More or less the same, the behaviour is not the same but it does serve the same purpose